CONTRIBUTING

```diff
@@ -0,0 +1,118 @@
+# Contributing to the CNTI
+
+Welcome to the Cloud Native Telecom Initiative (CNTi). This is an open, public project welcoming anyone who would like to help in one or more of our focus areas.  Our goal is to advance cloud native networking including developing best practices, maintaining a testing catalog, and providing vendor neutral certification.  We're glad you're here!
+
+To learn more about our focus areas and our different collaboration tools, please visit our focus area wiki pages:
+
+- [Best Practices](https://wiki.lfnetworking.org/x/HAAxBw)
+
+- [Test Catalog](https://wiki.lfnetworking.org/x/HgAxBw)
+
+- [Certification](https://wiki.lfnetworking.org/x/IAAxBw)
+
+## What to Contribute?
+
+We welcome many different types of contributions including:
+
+Best Practices:
+- General improvements to our documentation
+- [Use cases](doc/use-case/) and user stories (these are used to provide context for and select best practices)
+  - [See the template for more details](doc/use-case/NNNN-UC-template.md)
+- Definitions
+  - [Actors and Roles](https://github.com/lfn-cnti/bestpractices/discussions/30)
+  - [Glossary](doc/glossary.md)
+- [CNTi Best Practices](doc/best_cnf_dev.md)
+  - [Process to publish a CNTi Best Practice](doc/cbpps/cnti_best_practice_process.md)
+  - [See the template for more details](doc/cbpps/NNNN-cbpp-template.md)
+- Gap Analysis
+
+Test Catalog:
+- General improvements to our [documentation](https://github.com/cnti-testcatalog/testsuite/blob/main/docs/TEST_DOCUMENTATION.md)
+- [Code new tests, test enhancements, and bug fixes](https://github.com/cnti-testcatalog/testsuite/blob/main/docs/TEST_DOCUMENTATION.md)
+- Example CNFs
+- Bug reports
+
+
+Certification:
+
+- General improvements to our [documentation](https://github.com/lfn-cnti/certification)
+- Become a [reviewer](https://github.com/lfn-cnti/certification) and help with certification of vendors who successfully demonstrate adherence to cloud native best practices
+
+Not sure if you can contribute with anything above?  Feel free to join our meetings and discussions anyway.  There are always new ways to contribute to our growing community.  
+
+
+## How to Contribute
+
+### Join our Meetings
+
+Absolutely everyone is welcome to come to any of our meetings. You never need an invite to join us. In fact, we want you to join us, even if you don’t have anything you feel like you can  contribute. Just being there is a good start!
+
+Over time, we hope that you feel comfortable voicing your opinions, ideas and providing feedback with the community. Sharing your own ideas, and experiences can help in ways you might not initially realize.
+
+Check the respective focus area wikis for more details on when we meet for [Best Practices](https://wiki.lfnetworking.org/x/HAAxBw), [Test Catalog](https://wiki.lfnetworking.org/x/HgAxBw), and [Certification](https://wiki.lfnetworking.org/x/IAAxBw). 
+
+### Join a GitHub Discussion
```

Comment:
additional section - "Join CNTi slack channels" ...?

Reply:
Another good idea, but I feel it should be sufficient with the links to the focus area wiki pages as they include the relevant Slack channels for each.